### DISCUSSION TITLE
Alert button is not visible. Installed the grafana with helm chart

### DISCUSSION THREAD

**Discussion by amitkpal83** [grafana/grafana#56723] [Alerting]
![image](https://user-images.githubusercontent.com/6175985/195114225-0fd7d760-123b-4ad1-890c-f112b8d9c685.png)
Can anybody help what i am doing wrong?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
